tools/dune_rule_gen/gen_rules.ml

```diff
@@ -74,7 +85,7 @@ let main () =
   (* Rule generation *)
   let dir_info = Dir_info.scan ~prefix:[] base_dir in
   let directory = Path.make base_dir in
-  let theory = Coq_rules.Theory.{ directory; dirname = tname; implicit } in
+  let theory = Coq_rules.Theory.{ directory; dirname = tname; implicit; deps = dependencies } in
```

Comment:
Please make the names of the records field uniform.

Reply:
If I use `deps` (the nice name) I get a warning about shadowing a field elsewhere.
So I opted for the ugly name locally here in the argument parsing, and the nice one in the `Theory.t` type declaration.

Frankly, compared to the previous 6-tuple, this record seems a win even if a field name does not match.